packages/launchpad/src/global/GlobalPage.vue

```diff
@@ -122,7 +122,7 @@ const props = defineProps<{
 }>()
 
 const filteredProjects = computed(() => {
-  return (props.gql.projects as GlobalProjectCardFragment[]).filter((p) => p.title.toLowerCase().indexOf(match.value.toLowerCase()) !== -1)
+  return ((props.gql.projects || []) as GlobalProjectCardFragment[]).filter((p) => p.title.toLowerCase().indexOf(match.value.toLowerCase()) !== -1)
```

Comment:
I think we can make this cleaner and more idiomatic from a TS and JS point of view with a type guard and using `includes` instead of `indexOf`. I created a PR against this branch, what do you think? https://github.com/cypress-io/cypress/pull/20381/files

Reply:
merged